backend/main.py

```diff
@@ -1,9 +1,16 @@
-from src.api.main import app
+import os
 import uvicorn
+from dotenv import load_dotenv
+
+from src.api.main import app
+
+load_dotenv()
 
 
 def main() -> None:
-    uvicorn.run(app, host='127.0.0.1', port=8000)
+    uvicorn.run(
+        app, host='127.0.0.1', port=8000, workers=int(os.getenv('BACKEND_WORKERS', 1))
```

Comment:
Why set this to be 4? Should this not be dependent on the number of cores the backend has?

Reply:
Yes, that's why its being read from an env var. I'll switch this to 1.